tests/integration/testutil/common.go

```diff
@@ -4,54 +4,59 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/require"
+	//"github.com/stretchr/testify/require"
```

Comment:
It seems like an unnecessary comment

Reply:
https://github.com/xpladev/xpla/pull/154/commits/36c43e858dbe279ed6b6323cdc53748b700db29f